src/scrapers/visa-cal.test.ts

```diff
@@ -50,4 +50,19 @@ describe('VisaCal legacy scraper', () => {
     // expect(result?.accounts?.length).toEqual(2)
     exportTransactions(COMPANY_ID, result.accounts || []);
   });
+
+  maybeTestCompanyAPI(COMPANY_ID)('should scrape future debits"', async () => {
+    const options = {
+      ...testsConfig.options,
+      companyId: COMPANY_ID,
+    };
+
+    const scraper = new VisaCalScraper(options);
+    const result = await scraper.scrape(testsConfig.credentials.visaCal);
+    expect(result).toBeDefined();
+    const error = `${result.errorType || ''} ${result.errorMessage || ''}`.trim();
+    expect(error).toBe('');
+    expect(result.success).toBeTruthy();
+    expect(result.futureDebits?.length).toBeGreaterThan(0);
```

Comment:
we can use the same test we use for transactions to check this one as well in line 52, no need to run the scrapping again. 

Also, because it is optional, I don't believe we should assume in the expect that we got results. wdyt?

Reply:
I don't think its optional for CAL, as it is expected to always have this value.
the reason it is marked as optional, is because not all scrappers implemented it (only CAL as this is the first time its implemented in this PR)

Reply:
![Screen Shot 2022-02-20 at 11 06 12](https://user-images.githubusercontent.com/2913322/154835813-65b532da-d0f6-45f9-bf79-8121fc639aaa.png)
that how mine is looking

Reply:
![Screen Shot 2022-02-20 at 11 06 12](https://user-images.githubusercontent.com/2913322/154835838-e5d6e6a3-517a-4856-9149-3785e4ff8123.png)
that how my screen looks

regarding moving the test- sure I can move it there

Reply:
Hi, @LiranBri. I double checked the settings page and I don't find a way to enable this widget that you have. So we must assume this is not mandatory and other people might not have this section like in my scenario. 

We can proceed with your changes once you remove the tests as they don't apply to anyone . Since tests are not part of the ci/cd, you can just have them on your machine locally without adding them to the source control. wdyt?

Reply:
sure, i get your point. I removed it.